src/lib/y2network/wicked/connection_config_writers/base.rb

```diff
@@ -40,17 +43,15 @@ def initialize(file)
         #
         # @param conn [Y2Network::ConnectionConfig::Base] Connection to take settings from
         def write(conn)
-          file.bootproto = conn.bootproto&.name
-          file.name = conn.description
-          file.lladdr = conn.lladdress
-          file.startmode = conn.startmode.to_s
-          file.dhclient_set_hostname = dhclient_set_hostname(conn)
+          file.bootproto = value_as_string(conn.bootproto&.name)
```

Comment:
np: Perhaps we might think about adding support for such a conversion to "bootproto=" (and the rest of these methods).

Reply:
+1 we have similar conversions (from string to a <type>) ... so it would be even nice to be consistent